azure-devops/javascript/stages/30-tests/yarn.yaml

```diff
@@ -13,3 +13,11 @@ stages:
               COMMANDS: 'yarn osd bootstrap && yarn test:coverage'
               ARTIFACT_NAME: 'coverage'
               ARTIFACT_PATH: 'coverage'
+
+      - job: 'run_e2e_tests'
+        displayName: 'Run E2E Tests'
+        steps:
+          - template: '../../../global/abstracts/script-e2e-tests.yml'
+            parameters:
+              TAG_NAME: '2.16.0'
```

Comment:
Can you please use this parameters to pass the ports? So, inside the task you can iterate the ports you had.